src/components/PriceModel.tsx

```diff
@@ -84,6 +84,52 @@ const MarkerText: FC<{ children: ReactNode; ratio: number }> = ({
   </div>
 );
 
+
+const CompanyMarkers: FC<typeof usePeRatios & { ETH: number, peRatios: any }> = ({ peRatios }) => {
+  const markerList: Array<typeof Marker | undefined> =
```

Comment:
Here also. The list is always a list of `Marker` I think. Never a list of `undefined`. It's true you may ask for an element at an index where there's nothing in there, in which case `undefined` comes out, but that doesn't make it a list of `undefined` I'd say. In any case, we use a strict config for TypeScript where it will itself return `Thing | undefined` whenever you get something from an array.

Why did you add the `undefined` here and below?

Reply:
Updated! Indeed works fine without.